src/main/resources/db/migration/V2_4__add_gosys_oppgave.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE VEDTAK ADD COLUMN oppgave_at TIMESTAMPTZ;
+ALTER TABLE VEDTAK ADD COLUMN oppgave_id VARCHAR(20);
+
+UPDATE VEDTAK SET oppgave_id='0';
```

Comment:
Legger inn en default-verdi for alle eksisterende vedtak slik at det ikke genereres oppgaver for dem.